src/drivers/github.go

```diff
@@ -24,3 +32,65 @@ func (driver GithubCodeHostingDriver) GetNewPullRequestURL(repository string, br
 func (driver GithubCodeHostingDriver) GetRepositoryURL(repository string) string {
 	return "https://github.com/" + repository
 }
+
+// MergePullRequest merges the pull request through the Github API
+func (driver GithubCodeHostingDriver) MergePullRequest(options MergePullRequestOptions) error {
+	ts := oauth2.StaticTokenSource(
+		&oauth2.Token{AccessToken: os.Getenv("GITHUB_TOKEN")},
+	)
+	tc := oauth2.NewClient(context.Background(), ts)
+	client := github.NewClient(tc)
+	err := driver.updatePullRequestsAgainst(client, options)
+	if err != nil {
+		return err
+	}
+	return driver.mergePullRequest(client, options)
+}
+
+// Helper
+
+func (driver GithubCodeHostingDriver) mergePullRequest(client *github.Client, options MergePullRequestOptions) error {
```

Comment:
Let's store the github.Client instance in the driver so that we don't need to provide it to all methods.